ci-operator/step-registry/hypershift/hostedcluster/create/hostedcluster/hypershift-hostedcluster-create-hostedcluster-commands.sh

```diff
@@ -46,28 +54,59 @@ if [[ "${COMPUTE_NODE_TYPE}" == "" ]]; then
 fi
 
 echo "$(date) Creating HyperShift cluster ${CLUSTER_NAME}"
-/usr/bin/hypershift create cluster aws \
-  ${EXTRA_ARGS} \
-  --name ${CLUSTER_NAME} \
-  --infra-id ${INFRA_ID} \
-  --node-pool-replicas ${HYPERSHIFT_NODE_COUNT} \
-  --instance-type=${COMPUTE_NODE_TYPE} \
-  --base-domain ${DOMAIN} \
-  --region ${HYPERSHIFT_AWS_REGION} \
-  --control-plane-availability-policy ${HYPERSHIFT_CP_AVAILABILITY_POLICY} \
-  --infra-availability-policy ${HYPERSHIFT_INFRA_AVAILABILITY_POLICY} \
-  --pull-secret=/tmp/pull-secret.json \
-  --aws-creds=${AWS_GUEST_INFRA_CREDENTIALS_FILE} \
-  --release-image ${RELEASE_IMAGE} \
-  --control-plane-operator-image=${CONTROLPLANE_OPERATOR_IMAGE:-} \
-  --node-selector "hypershift.openshift.io/control-plane=true" \
-  --additional-tags="expirationDate=$(date -d '4 hours' --iso=minutes --utc)" \
-  --annotations "prow.k8s.io/job=${JOB_NAME}" \
-  --annotations "prow.k8s.io/build-id=${BUILD_ID}" \
-  --annotations resource-request-override.hypershift.openshift.io/kube-apiserver.kube-apiserver=memory=3Gi,cpu=2000m \
-  --annotations hypershift.openshift.io/cleanup-cloud-resources="false" \
-  --additional-tags "prow.k8s.io/job=${JOB_NAME}" \
-  --additional-tags "prow.k8s.io/build-id=${BUILD_ID}"
+case "${PLATFORM}" in
+  "aws")
+    /usr/bin/hypershift create cluster aws \
+      ${EXTRA_ARGS} \
+      --name ${CLUSTER_NAME} \
+      --infra-id ${INFRA_ID} \
+      --node-pool-replicas ${HYPERSHIFT_NODE_COUNT} \
+      --instance-type=${COMPUTE_NODE_TYPE} \
+      --base-domain ${DOMAIN} \
+      --region ${HYPERSHIFT_AWS_REGION} \
+      --control-plane-availability-policy ${HYPERSHIFT_CP_AVAILABILITY_POLICY} \
+      --infra-availability-policy ${HYPERSHIFT_INFRA_AVAILABILITY_POLICY} \
+      --pull-secret=/tmp/pull-secret.json \
+      --aws-creds=${AWS_GUEST_INFRA_CREDENTIALS_FILE} \
+      --release-image ${RELEASE_IMAGE} \
+      --control-plane-operator-image=${CONTROLPLANE_OPERATOR_IMAGE:-} \
```

Comment:
do you wanna use this param for the powervs platform as well?

Reply:
yes, used.